x/evm/keeper/migrate.go

```diff
@@ -49,6 +52,12 @@ func removeExternalTokenBurnerCode(ctx sdk.Context, ck chainKeeper) error {
 	return nil
 }
 
+func setEndBlockerLimitParam(ctx sdk.Context, ck chainKeeper) {
+	params := ck.GetParams(ctx)
```

Comment:
I'm 85% sure that this would cause a panic. Check https://github.com/axelarnetwork/axelar-core/blob/ccd0c55fc39b6cfb5232d4ceb89da9f14328c459/x/nexus/keeper/migrate.go#L70 for the proper way of adding new parameters in migration.

Reply:
not panic in test, but yes, the other way is better

Reply:
It didn't panic in test only because https://github.com/axelarnetwork/axelar-core/blob/c57017508c6ed9e783cb0f87e4d7acf3d45f2431/x/evm/keeper/migrate_test.go#L29 set the latest version of the params with latest param set. It won't be the case for the actual migration.